.github/workflows/ci.yml

```diff
@@ -33,7 +33,8 @@ jobs:
       - name: Install dependencies
         shell: bash -l {0}
         run: |
-          conda install -c conda-forge pip botocore aiobotocore moto pytest flake8 black -y
+          conda install -c conda-forge pip botocore aiobotocore flake8 black -y
```

Comment:
why not include the moto version pin in the conda install line? pip could in theory step on conda packages, so we should keep conda-based tests and pip-based tests separate.

Reply:
I just thought maintaining the same list in 2 different places was redundant, though can revert.

Reply:
True, but I don't see a way around it when I trust conda more than pip

Reply:
alright dropped